torch/csrc/distributed/c10d/init.cpp

```diff
@@ -1756,7 +1756,16 @@ The hook must have the following signature:
           .def(
               "_has_hooks",
               &::c10d::ProcessGroup::hasHooks,
-              py::call_guard<py::gil_scoped_acquire>());
+              py::call_guard<py::gil_scoped_acquire>())
+          .def(
+              "_set_group_name",
+              &::c10d::ProcessGroup::setGroupName,
+              py::call_guard<py::gil_scoped_acquire>(),
+              "Sets the process group name. This is an internal C10D method, do not use.")
+          .def_property_readonly(
+              "group_name",
+              &::c10d::ProcessGroup::getGroupName,
+              "(Gets this process group name. It's cluster unique)");
```

Comment:
Do we log this already somewhere?

Reply:
We are already logging the group_name in the scuba table. So once the "None" issue is fixed, the correct group_name should be populated into our table.

Reply:
This PR unfortunately doesn't fix the issue but will help us troubleshoot it.